src/code/audio_sound_params.c

```diff
@@ -1,238 +1,1290 @@
+    SFX_PARAMS(0x40, 1, 0, 0),            // NA_SE_EN_BARI_SPLIT
+    SFX_PARAMS(0x28, 1, 0, 0),            // NA_SE_EN_TEKU_REVERSE
+    SFX_PARAMS(0x30, 3, 0, 0),            // NA_SE_EN_VALVAISA_LAND2
+    SFX_PARAMS(0x20, 0, 0, 0),            // NA_SE_EN_TEKU_LAND_WATER
+    SFX_PARAMS(0x38, 1, 0, 0),            // NA_SE_EN_LAST_DAMAGE
+    SFX_PARAMS(0x30, 0, 0, 0),            // NA_SE_EN_STALWALL_ROLL
+    SFX_PARAMS(0x30, 0, 0, 0),            // NA_SE_EN_STALWALL_DASH
+    SFX_PARAMS(0x20, 0, 0, 0),            // NA_SE_EN_TEKU_JUMP_WATER
+    SFX_PARAMS(0x20, 0, 0, 0),            // NA_SE_EN_TEKU_LAND_WATER2
+    SFX_PARAMS(0x38, 0, 0, SFX_FLAG_13),  // NA_SE_EN_FALL_AIM
+    SFX_PARAMS(0x30, 3, 0, 0),            // NA_SE_EN_FALL_UP
+    SFX_PARAMS(0x30, 0, 0, SFX_FLAG_13),  // NA_SE_EN_FALL_CATCH
+    SFX_PARAMS(0x30, 0, 0, 0),            // NA_SE_EN_FALL_LAND
+    SFX_PARAMS(0x14, 0, 0, 0),            // NA_SE_EN_FALL_WALK
+    SFX_PARAMS(0x38, 1, 0, 0),            // NA_SE_EN_FALL_DAMAGE
+    SFX_PARAMS(0x40, 1, 0, 0),            // NA_SE_EN_FALL_DEAD
+    SFX_PARAMS(0x14, 0, 0, 0),            // NA_SE_EN_KAICHO_FLUTTER
+    SFX_PARAMS(0x20, 0, 0, 0),            // NA_SE_EN_BIRI_FLY
+    SFX_PARAMS(0x20, 0, 0, 0),            // NA_SE_EN_BIRI_JUMP
+    SFX_PARAMS(0x30, 0, 0, 0),            // NA_SE_EN_BIRI_SPARK
+    SFX_PARAMS(0x40, 1, 0, 0),            // NA_SE_EN_BIRI_DEAD
+    SFX_PARAMS(0x40, 1, 0, 0),            // NA_SE_EN_BIRI_BUBLE
+    SFX_PARAMS(0x30, 0, 0, 0),            // NA_SE_EN_BARI_ROLL
+    SFX_PARAMS(0x34, 1, 0, 0),            // NA_SE_EN_GOMA_JR_FREEZE
+    SFX_PARAMS(0x40, 1, 0, 0),            // NA_SE_EN_BARI_DEAD
+    SFX_PARAMS(0x30, 3, 0, 0),            // NA_SE_EN_GANON_FIRE
+    SFX_PARAMS(0x30, 2, 0, 0),            // NA_SE_EN_FANTOM_TRANSFORM
+    SFX_PARAMS(0x30, 0, 0, SFX_FLAG_13),  // NA_SE_EN_FANTOM_THUNDER
+    SFX_PARAMS(0x20, 3, 1, 0),            // NA_SE_EN_FANTOM_SPARK
+    SFX_PARAMS(0x20, 2, 0, 0),            // NA_SE_EN_FANTOM_FLOAT
+    SFX_PARAMS(0x30, 3, 0, 0),            // NA_SE_EN_FANTOM_MASIC1
+    SFX_PARAMS(0x30, 3, 0, 0),            // NA_SE_EN_FANTOM_MASIC2
+    SFX_PARAMS(0x30, 3, 0, 0),            // NA_SE_EN_FANTOM_FIRE
+    SFX_PARAMS(0x38, 3, 0, 0),            // NA_SE_EN_FANTOM_HIT_THUNDER
+    SFX_PARAMS(0x30, 0, 0, SFX_FLAG_13),  // NA_SE_EN_FANTOM_ATTACK
+    SFX_PARAMS(0x30, 3, 0, 0),            // NA_SE_EN_FANTOM_STICK
+    SFX_PARAMS(0x30, 2, 0, 0),            // NA_SE_EN_FANTOM_EYE
+    SFX_PARAMS(0x30, 0, 0, SFX_FLAG_13),  // NA_SE_EN_FANTOM_LAST
+    SFX_PARAMS(0x30, 3, 0, 0),            // NA_SE_EN_FANTOM_THUNDER_GND
+    SFX_PARAMS(0x38, 3, 0, 0),            // NA_SE_EN_FANTOM_DAMAGE
+    SFX_PARAMS(0x40, 3, 0, 0),            // NA_SE_EN_FANTOM_DEAD
+    SFX_PARAMS(0x30, 3, 0, 0),            // NA_SE_EN_FANTOM_LAUGH
+    SFX_PARAMS(0x30, 3, 0, 0),            // NA_SE_EN_FANTOM_DAMAGE2
+    SFX_PARAMS(0x30, 3, 0, SFX_PARAM_10), // NA_SE_EN_FANTOM_VOICE
```

Comment:
Well now this looks weird lol :sweat_smile: , having a "param" as a flag

(I can live with it)

Reply:
Yeah I think treating it as a param would only make sense if we moved it outside of flags entirely, and used shift/mask defines like the other params. But I don't know if it's an improvement over just treating it as a flag like it was before (probably not tbh)

Reply:
we could leave this as is and come back to it when this stuff is actually documented

Reply:
I think we should commit one way or the other. So either:

as it was:
`SFX_PARAMS(0x30, 3, 0, SFX_FLAG_10), // NA_SE_EN_FANTOM_VOICE`

as an param (index) with an extra arg for all `SFX_PARAMS`:
`SFX_PARAMS(0x30, 3, 0, 1, 0), // NA_SE_EN_FANTOM_VOICE`

Perhaps?

Reply:
I don't really care tbh (it can change later as needed), but if you need an opinion to make up your mind I myself think a flag is enough, but happy to accept the other if @Roman971 or someone prefers it

Reply:
Yeah I think it's fine to keep it as a flag (ie. as it was), then come back to it when it's documented and see if it makes more sense as a param